ext/groonga/rb-grn-table.c

```diff
@@ -395,6 +400,8 @@ rb_grn_table_define_index_column (int argc, VALUE *argv, VALUE self)
                         "with_section", &rb_with_section,
                         "with_weight", &rb_with_weight,
                         "with_position", &rb_with_position,
+                        "index_small", &rb_index_small,
```

Comment:
`:size => :small` or `:size => :medium` API is better. Because `:index_small` and `:index_medium` are exclusive. Users can't specify both small and medium by `:size => ...` API unexpectedly.